App/Screens/About/About.js

```diff
@@ -16,15 +16,33 @@
 
 import React, { PureComponent } from 'react';
 import Constants from 'expo-constants';
-import { Linking, Platform, ScrollView, StyleSheet, Text, View } from 'react-native';
+import {
+  Linking,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
 
 import { Box } from './Box';
 import { BackButton } from '../../components/BackButton';
 import { Language } from './Language';
 import { i18n } from '../../localization';
 import * as theme from '../../utils/theme';
 
+import { getData } from '../../managers/AqiHistoryDb';
+
 export class About extends PureComponent {
+  state = {
+    history: ''
+  };
+
+  async componentDidMount () {
+    const history = await getData(10);
+    this.setState({ history });
```

Comment:
This was actually only for testing purpose, I would prefer this not to go in master

Reply:
Oh 😄 Sure!